mysql-test/suite/rocksdb/include/locking_issues_case1_1.inc

```diff
@@ -31,10 +31,10 @@ SELECT * FROM t0 WHERE id1=1 AND id2=5 FOR UPDATE;
 connection con2;
 eval SET SESSION TRANSACTION ISOLATION LEVEL $isolation_level;
 BEGIN;
---error ER_LOCK_WAIT_TIMEOUT
+--error ER_QUERY_TIMEOUT
```

Comment:
This and other re-recorded tests. I'm not sure this is OK. This may be the side effect of the fix which loses information about lock_wait_timeout. Now every ER_LOCK_WAIT_TIMEOUT is converted into ER_QUERY_TIMEOUT which I think is not OK. We need to distinguish these 2 cases like we do for InnoDB